### PR TITLE
Refactor topic renaming slightly to use TableId instead of two maps

### PR DESCRIPTION
**Pull request by criccomini**
Did several things in this PR:

* Refactored topic rename from #44 to remove some duplicate maps.
* Cleaned some checkstyle issues
* Fixed a 404 error with some javadoc links

### CODE REVIEW

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkConnector.java [hunk 1]
```diff
@@ -120,33 +117,25 @@ private void ensureExistingTables(
...
-      TableId tableId = TableId.of(dataset, table);
+      Map<String, TableId> topicsToTableIds) {
+    for (TableId tableId : topicsToTableIds.values()) {
       if (bigQuery.getTable(tableId) == null) {
         logger.warn(
-              "You may want to enable auto table creation by specifying"
-              + "{}=true in the properties file",
-              config.TABLE_CREATE_CONFIG
-        );
+            "You may want to enable auto table creation by setting {}=true in the properties file",
```
Comment (mtagle) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkConnector.java]: isn't this line too long?

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/TopicToTableResolver.java [hunk 2]
```diff
@@ -34,75 +37,79 @@
...
+  }
+
+  /**
+   * Strips illegal characters from a table name. BigQuery only allows alpha-numeric and
+   * underscore. Everything illegal is converted to an underscore.
+   *
+   * @param tableName The table name to sanitize.
+   * @return A clean table name with only alpha-numerics and underscores.
+   */
+  public static String sanitizeTableName(String tableName) {
```
Comment (mtagle) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/TopicToTableResolver.java]: why does this need to be public?

File: kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTaskTest.java [hunk 3]
```diff
@@ -44,6 +44,7 @@
 import com.wepay.kafka.connect.bigquery.exception.BigQueryConnectException;
 import com.wepay.kafka.connect.bigquery.exception.SinkConfigConnectException;
 
+import com.wepay.kafka.connect.bigquery.utils.TopicToTableResolver;
```
Comment (mtagle) [inline kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTaskTest.java]: is this necessary?

Comment (criccomini) [inline kcbq-connector/src/test/java/com/wepay/kafka/connect/bigquery/BigQuerySinkTaskTest.java]: Good point.

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/TopicToTableResolver.java (hunk 2 above)
Comment (criccomini) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/utils/TopicToTableResolver.java]: Just seemed generally useful. I can swap back to private, though. Nothing else is using it.

File: kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkConnector.java (hunk 1 above)
Comment (criccomini) [inline kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/BigQuerySinkConnector.java]: It's exactly 100 chars.. :D

**Pull request merged**